src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -82,6 +84,12 @@ class RemoteMachineTrainingService implements TrainingService {
         restServer.setEnableVersionCheck = this.versionCheck;
         this.log.info('Run remote machine training service.');
         while (!this.stopping) {
+            if (this.sshConnectionPromises.length) {
```

Comment:
To improve readabilty, and avoid tricks of JS, it's bettter to write like `this.sshConnectionPromises.length > 0`

Reply:
fixed